hack/istio/multicluster/install-multi-primary.sh

```diff
@@ -242,8 +242,8 @@ source ${SCRIPT_DIR}/split-bookinfo.sh
 # Install Kiali if enabled
 if [ "${KIALI_ENABLED}" == "true" ]; then
   if [ -z "${KEYCLOAK_ADDRESS}" ]; then
-    echo "Keycloak is not available for this cluster setup. Switching Kial to 'anonymous' mode."
-    export KIALI_AUTH_STRATEGY="anonymous"
+    echo "Keycloak is not available for this cluster setup. Switching Kiali to 'anonymous' mode."
+    # export KIALI_AUTH_STRATEGY="anonymous"
```

Comment:
I remember having to add this very recently. Are you sure we do not need this anymore?

Reply:
Yes because we don't want `anonymous` we want the auth strategy we passed in which is `openshift`. This should be commented out so I'll look at cleaning it up. Why was it changed to `anonymous` instead of the passed in auth strategy?